foundry.toml

```diff
@@ -7,7 +7,7 @@ test='test'
 out='out'
 libs=['lib']
 cache_path='cache'
-fs_permissions = [{ access = "read-write", path = "./addresses/"}, { access = "read", path = "./out/" }]
+fs_permissions = [{ access = "read-write", path = "./addresses/"}, { access = "read-write", path = "./script/core/monitors/"}, { access = "read", path = "./out/" }]
```

Comment:
A directory outside of the source subtrees would be nice, in particular because those directories are not ignored by git. Maybe `./monitor_results` and add `monitor_results` to `.gitignore`?